moped-editor/src/utils/activityLogFormatters/mopedTagsActivity.js

```diff
@@ -2,18 +2,30 @@ import LocalOfferOutlinedIcon from "@material-ui/icons/LocalOfferOutlined";
 
 export const formatTagsActivity = (change, tagList) => {
   const changeIcon = <LocalOfferOutlinedIcon />;
-  let changeDescription = "Project tags updated";
-  let changeValue = ""
 
   // Adding a new tag
   if (change.description.length === 0) {
-    changeDescription = "Project tagged with "
-    changeValue = tagList[change.record_data.event.data.new.tag_id]
+    return {
+      changeIcon,
+      changeText: [
+        { text: "Project tagged with ", style: null },
+        {
+          text: tagList[change.record_data.event.data.new.tag_id],
+          style: "boldText",
+        },
+      ],
+    };
   } else {
     // Soft deleting a tag is the only update a user can do (is_deleted is set to true)
-    changeDescription = "Project tag deleted "
-    changeValue = tagList[change.record_data.event.data.new.tag_id]
+    return {
+      changeIcon,
+      changeText: [
+        { text: "Project tag deleted  ", style: null },
```

Comment:
this could now be "Project tag TAGNAME was deleted" 

as long as we are okay with passive voice